modules/forms/src/components/scopes.tsx

```diff
@@ -19,11 +19,12 @@
 import filter from "lodash-es/filter";
 import isEmpty from "lodash-es/isEmpty";
 import isEqual from "lodash-es/isEqual";
+import { TestableComponentInterface } from "modules/core/dist/types/models";
```

Comment:
Import from the module.

```suggestion
import { TestableComponentInterface } from "@wso2is/core/models";
```